src/utils/DropdownButton.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+    <div :class="{ 'btn-group': withToggleButton }">
+        <button
+            :id="withToggleButton ? null : uniqueHtmlId"
+            ref="dropdownMainButton"
+            :disabled="disabled"
+            class="btn btn-outline-light text-dark"
+            :class="{ 'dropdown-toggle': !withToggleButton }"
+            type="button"
+            data-cy="dropdown-main-button"
+            :data-bs-toggle="withToggleButton ? null : 'dropdown'"
+            :aria-expanded="withToggleButton ? null : expanded"
+            @click="onMainButtonClick"
+        >
+            {{ $t(title) }}
+        </button>
+        <button
+            v-if="withToggleButton"
+            :id="uniqueHtmlId"
+            ref="dropdownToggleButton"
+            :disabled="disabled"
+            type="button"
+            class="btn btn-outline-light text-dark dropdown-toggle dropdown-toggle-split"
+            data-cy="dropdown-toggle-button"
+            data-bs-toggle="dropdown"
+            data-bs-reference="parent"
+            :aria-expanded="expanded"
+            @click="toggleExpanded"
+        >
+            <span class="visually-hidden">Toggle Dropdown</span>
+        </button>
+        <ul class="dropdown-menu" :class="{ show: expanded }" :aria-labelledby="uniqueHtmlId">
+            <li v-for="item in items" :key="item.value">
+                <a
+                    class="dropdown-item"
```

Comment:
It seems that there is an issue with the `dropdown-item` css class. The cursor onhover on a dropdown item is not set to `pointer`